src/components/Table/GridTable.tsx

```diff
@@ -556,12 +557,12 @@ function renderVirtual<R extends Kinded>(
 
         // If it is a chrome row, then we are not using `display: contents;`, return the height of this element.
         if ("chrome" in maybeContentsDiv.dataset) {
-          return maybeContentsDiv.offsetHeight;
+          return maybeContentsDiv.getBoundingClientRect().height || 1;
         }
 
         // Both the `Item` and `itemContent` use `display: contents`, so their height is 0,
         // so instead drill into the 1st real content cell.
-        return (maybeContentsDiv.firstElementChild! as HTMLElement).offsetHeight;
+        return (maybeContentsDiv.firstElementChild! as HTMLElement).getBoundingClientRect().height;
```

Comment:
**question:** Is the `...clientRect` returns a float vs an integer?

Reply:
Yep!